### DISCUSSION TITLE
Junior .Net Developer z Azure

### DISCUSSION THREAD

**Discussion by Daniil1313** [AzureCommunityPL/group#481] [Oferty pracy]
**AXA IT Solutions**

Jesteśmy wewnętrznym software housem działającym w ramach międzynarodowej grupy ubezpieczeniowej AXA. Dostarczamy rozwiązania informatyczne na potrzeby spółek AXA w Europie. Pracujemy na co dzień w języku angielskim, realizując międzynarodowe projekty deweloperskie. Nasze biuro znajdziesz na ul. Chłodnej 51 w Warszawie. Stale się uczymy, dzielimy wiedzą i rozwijamy, powiększając nasz zespół oraz realizując kolejne projekty IT.

Z NAMI BĘDZIESZ:

- Pracować w pełnym, dojrzałym cyklu tworzenia oprogramowania (zbieranie wymagań, projektowanie/prototypowanie rozwiązań i konsultacja z klientem, pisanie kodu, wdrażanie, testowanie) zgodnym z podejściem Agile.
- Tworzyć testy weryfikujące wytworzone oprogramowanie (testy jednostkowe, testy integracyjne)
- Wykorzystywać nowinki techniczne i obecne trendy do projektowania rozwiązań
- Współpracować z klientem biznesowym oraz innymi członkami zespołu projektowego
- Współpracować z zespołami w UK odpowiedzialnymi za definiowanie wymagań, tworzenie i dostarczanie rozwiązań backendowych

OFERUJEMY:

- Ciekawe, ambitne i różnorodne zadania przy realizacji projektu dla klienta biznesowego z UK
- Możliwość rozwoju zawodowego w międzynarodowym środowisku biznesowym i technicznym
- Atrakcyjny system bonusowy (prywatna opieka medyczna, multisport, konferencje, szkolenia, dofinansowanie nauki języków obcych)
- Nielimitowany dostęp do platformy szkoleniowej Pluralsight
- Pracę w dobrej atmosferze, wspierających się zespołach, wśród pracowników, którzy chętnie dzielą się wiedzą m.in. w ramach wewnętrznych inicjatyw rozwojowych

OCZEKIWANIA:

- Masz doświadczenie w tworzeniu i implementacji aplikacji napisanych w .NET, .NET Core
- Mile widziane doświadczenie z Azure
- Potrafisz tworzyć rozwiązania od zera, projektować i wyznaczać kierunek rozwoju aplikacji
- Znasz najlepsze praktyki programowania i wzorce projektowe
- Lubisz dzielić się wiedzą i doświadczeniem z innymi członkami zespołu
- Mówisz swobodnie po angielsku (min. B2)

Jeśli chcielibyście dowiedzieć się więcej, zapraszam na priv: https://www.linkedin.com/in/daniil-dubyna-61044622b/

Tryb pracy: hybrydowy
Lokalizacja: Warszawa
Widełki: 60-105 zł/g przy B2B. Przy UoP 7000 - 11 500 zł brutto

**Comment by aryll** [comment]
@Daniil1313 proszę uzupełnij brakujące dane: post MUSI zawierać: kwote/widełki, job description, forma współpracy, lokalizacja, office/remote/hybrid.
